src/FakeItEasy.Extensions.ValueTask/FakeItEasy.Extensions.ValueTask.csproj

```diff
@@ -30,7 +30,7 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' == 'net45' OR '$(TargetFramework)' == 'netstandard2.0'">
-    <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.5.3" />
+    <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.5.4" />

```

Comment:
Arguably out of scope, and I can drop (or split out to another PR), but we have upgraded some other dependencies as part of 7.0, and Microsoft.CodeAnalysis.CSharp 3.8.0 depends on it. I thought cleaner just to align, but whatever you like.

Reply:
Sorry, had committer's regret right after I shut the computer off for the night. Previous version updates have been for internal tools only. If we want to bump this dependency, it's more appropriate in a separate PR, which we could merge independently. Since it looks like I've beaten the reviewers in, I'm going to drop the commit.